tools/dependencies/known-dependencies.txt

```diff
@@ -50,6 +50,10 @@ aws-java-sdk-core-1.12.37.jar
 aws-java-sdk-ec2-1.12.37.jar
 aws-java-sdk-kms-1.12.37.jar
 aws-java-sdk-s3-1.12.37.jar
+bcpkix-jdk15on-1.68.jar
+bcprov-ext-jdk15on-1.68.jar
+bcprov-jdk15on-1.68.jar
+bouncy-castle-bc-2.8.0-pkg.jar
```

Comment:
Are these licenses also lost? I don't found in https://github.com/apache/incubator-seatunnel/pull/1984/files#diff-01f40f6b66e748f1845983d2ebae82ebcdc07c318bb0ee5ddab1fad666a2f859

Reply:
`bouncy-castle-bc-2.8.0-pkg.jar` is module of the `apache pulsar` project.
**LICENSE-bouncycastle.txt** is the license file for other dependencies.

Reply:
bcpkix-jdk15on-1.68 and bcprov-ext-jdk15on-1.68 etc...  is not reflected in https://github.com/apache/incubator-seatunnel/pull/1984/files#diff-01f40f6b66e748f1845983d2ebae82ebcdc07c318bb0ee5ddab1fad666a2f859 , could check again?

Reply:
I added bouncycastle's dependencies to the MIT license.